src/main/drivers/timer_common.c

```diff
@@ -77,15 +66,28 @@ const timerHardware_t *timerGetByTagAndIndex(ioTag_t ioTag, unsigned timerIndex)
     return NULL;
 }
 
+#else
+
+const timerHardware_t *timerGetByTagOnly(ioTag_t ioTag)
```

Comment:
This can be renamed to `timerGetByTag(ioTag_t ioTag)` and directly exposed if the below implementation of `timerGetByTag(ioTag_t ioTag)` is moved up into the `#if defined(USE_TIMER_MGMT)` block.